sdk/examples/how_tos/native_tokens/burn.rs

```diff
@@ -49,7 +49,7 @@ async fn main() -> Result<()> {
         .unwrap_or_else(|| TokenId::from(*balance.foundries().first().unwrap()));
 
     if let Some(native_token_balance) = balance.native_tokens().iter().find(|native_token| {
-        native_token.token_id() == &token_id && native_token.available() >= U256::from(MIN_AVAILABLE_AMOUNT)
+        native_token.token_id() == &token_id && native_token.available().as_u64() >= MIN_AVAILABLE_AMOUNT
```

Comment:
This can panic

Reply:
Ah yeah right. It's not an issue for tests because we don't explicitly use large values, but for examples maybe it's a problem.

Reply:
I'll just change everywhere

Reply:
Yeah deleted my comment for the test since we set the values there

Reply:
Removed